benchmark-backend/pom.xml

```diff
@@ -61,6 +61,23 @@
             <groupId>org.flywaydb</groupId>
             <artifactId>flyway-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+            <version>2.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.modelmapper</groupId>
+            <artifactId>modelmapper</artifactId>
+            <version>2.3.8</version>
+        </dependency>
```

Comment:
we already have objectmapper from jackson library. Is it required to use another one?

Reply:
Your're right, I already changed it in the next commit.